spec/javascript/packages/document-capture/components/document-capture-warning-spec.jsx

```diff
@@ -51,20 +55,39 @@ describe('DocumentCaptureWarning', () => {
       },
     ];
     return render(
-      <InPersonContext.Provider value={{ inPersonURL: inPersonUrl }}>
-        <DocumentCaptureWarning
-          isFailedDocType={isFailedDocType}
-          isFailedResult={isFailedResult}
-          remainingAttempts={2}
-          unknownFieldErrors={unknownFieldErrors}
-          actionOnClick={() => {}}
-        />
-        ,
-      </InPersonContext.Provider>,
+      <AnalyticsContext.Provider value={{ trackEvent }}>
+        <InPersonContext.Provider value={{ inPersonURL: inPersonUrl }}>
+          <DocumentCaptureWarning
+            isFailedDocType={isFailedDocType}
+            isFailedResult={isFailedResult}
+            remainingAttempts={2}
+            unknownFieldErrors={unknownFieldErrors}
+            actionOnClick={() => {}}
+          />
+          ,
+        </InPersonContext.Provider>
+      </AnalyticsContext.Provider>,
     );
   }
+
   context('ipp ', () => {
     const inPersonUrl = '/verify/doc_capture';
+
+    it('logs the warning displayed to the user', () => {
+      const isFailedResult = true;
+      const isFailedDocType = false;
+
+      renderCcontent(isFailedDocType, isFailedResult, inPersonUrl);
+
+      expect(trackEvent).to.have.been.calledWith('IdV: warning shown', {
+        location: 'doc_auth_review_issues',
+        heading: 'errors.doc_auth.rate_limited_heading',
+        subHeading: '',
```

Comment:
Could we include at least one spec which demonstrates that the subheading is logged?

Reply:
Great suggestion! Fixed in aa67250c4994bd50bcd38778f45167d2df66a533.